src/components/Modals/Onboarding.vue

```diff
@@ -0,0 +1,742 @@
+<template>
+  <div class="onboarding">
+    <div class="onboarding-main">
+      <div class="onboarding-content">
+        <div class="onboarding-top">
+          <div v-if="!isEditingProfile" class="onboarding-header">👏 Welcome to CoursePlan</div>
+          <div v-if="isEditingProfile" class="onboarding-header">👋 Hi {{ user.firstName }}</div>
```

Comment:
Be careful about using emojis in UI. They look different in different OS!

Reply:
Thanks for the heads up